### DISCUSSION TITLE
Bloatynosy 1.2

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Bloatynosy#112] [Announcements]
Bloaty**N**osy has been renamed to **Bloatynosy** ("the 'n' in 'nosy' is written as one word in lowercase.). Due to the renaming of the GitHub repository, the update mechanism is currently unfortunately disrupted. 
- Numerous optimizations and bug fixes, along with the aftermath of the final release, have been resolved.

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Bloatynosy/releases/tag/1.2.12'>Bloatynosy 1.2</a>.</em>

**Comment by DealsBeam** [comment]
Terrible name but as long as its functional who cares

**Reply by Belim** [reply]
the name has won numerous awards
